scripts/ci/update-github

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -ex
```

Comment:
It won't clutter the changelog because the changelog is not taking everything printed to console. It's just taking everything assigned to `MARKDOWN` variable.

Reply:
For debugging in jenkins it's nice to have the -x flag!